torchgeo/datasets/patternnet.py

```diff
@@ -79,6 +80,47 @@ class PatternNet(VisionClassificationDataset):
     filename = "PatternNet.zip"
     directory = "images"
 
+    label_dict = {
```

Comment:
In other datasets we make this a list of label strings called "classes". We should do the same here.

Reply:
What I meant is you should just remove the classes attribute you added because VisionClassificationDataset already has one that you can use to convert int->str labels.